applications/base/skills.py

```
@@ -309,76 +602,170 @@ class move_to_pose_definition(BaseModel):
+            target_pose = goal_pose
+            
+        # State 4: Execute cartesian movement
+        try:
+            response = robots[robot_to_use - 1].move_to_pose(target_pose, tool_vel=tool_vel, tool_acc=tool_acc, user_num=user_num)
+            if response:
+                print(f"Cartesian motion SUCCEEDED: target pose - {target_pose}")
+            else:
+                send_message_to_webapp_implementation()._run("Cartesian movement failed - robot returned failure")
+                return False
+                
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)